packages/integration-sdk-runtime/src/storage/FileSystemGraphObjectStore/FileSystemGraphObjectStore.ts

```diff
@@ -153,11 +152,17 @@ export class FileSystemGraphObjectStore implements GraphObjectStore {
         integrationStepsToGraphObjectIndexMetadataMap(params.integrationSteps);
     }
   }
+  getStepsStored() {
+    return this.localGraphObjectStore.getStepsStored();
+  }
```

Comment:
Can you point me where is this used and what is this used for? I don't see any code referencing this new method.

Reply:
Thanks! It was used in a previous version of this change and is no longer needed. Is now removed!